runway/utils.py

```diff
@@ -23,6 +23,19 @@
         r'(?::\d+)?' # optional port
         r'(?:/?|[/?]\S+)$', re.IGNORECASE)
 
+def validate_post_request_body_is_json(f):
+    @functools.wraps(f)
+    def wrapped(*args, **kwargs):
+        json = get_json_or_none_if_invalid(request)
+        if json is not None:
+            return f(*args, **kwargs)
+        else:
+            err_msg = 'The body of all POST requests must contain JSON'
+            return jsonify(dict(error=err_msg)), 400
+    return wrapped
```

Comment:
A decorator to wrap around `POST` endpoints to ensure that requests that are sent to them contain JSON bodies. If they don't we immediately return a JSON error message and a 400 error code.